test/test_simple_commands.py

```diff
@@ -30,11 +30,11 @@
 # Expected nested page source
 NESTED_TEST_DIR = "/recursive_iframes/"
 NESTED_FRAMES_URL = utilities.BASE_TEST_URL + NESTED_TEST_DIR + "parent.html"
-D1_BASE = "http://d1." + utilities.BASE_TEST_URL_NOSCHEME + NESTED_TEST_DIR
-D2_BASE = "http://d2." + utilities.BASE_TEST_URL_NOSCHEME + NESTED_TEST_DIR
-D3_BASE = "http://d3." + utilities.BASE_TEST_URL_NOSCHEME + NESTED_TEST_DIR
-D4_BASE = "http://d4." + utilities.BASE_TEST_URL_NOSCHEME + NESTED_TEST_DIR
-D5_BASE = "http://d5." + utilities.BASE_TEST_URL_NOSCHEME + NESTED_TEST_DIR
+D1_BASE = utilities.BASE_TEST_URL + NESTED_TEST_DIR
+D2_BASE = utilities.BASE_TEST_URL + NESTED_TEST_DIR
+D3_BASE = utilities.BASE_TEST_URL + NESTED_TEST_DIR
+D4_BASE = utilities.BASE_TEST_URL + NESTED_TEST_DIR
+D5_BASE = utilities.BASE_TEST_URL + NESTED_TEST_DIR
```

Comment:
This is now pretty pointless as the different levels of iframe no longer belong to different domains. But I hope to find a way to restore this functionality either with a custom DNS server or some other way.

Reply:
Could you please open an issue about that so we don't forget?

Reply:
Created #887 and linked it in the file as well.